helm/harbor-scanner-trivy/templates/statefulset.yaml

```diff
@@ -43,27 +43,27 @@ spec:
           {{- end }}
           env:
             - name: "SCANNER_LOG_LEVEL"
-              value: {{ .Values.scanner.logLevel | quote }}
+              value: {{ .Values.scanner.logLevel | default "info" | quote }}
             - name: "SCANNER_API_SERVER_ADDR"
-              value: ":{{ .Values.service.port }}"
+              value: ":{{ .Values.service.port | default "8080" }}"
```

Comment:
Does this need `""`?

Reply:
We can always confirm such thing with helm template command. I did test it and `":{{ .Values.service.port | default 8080 }}"` works fine. I've also update the template